pkgs/servers/pulseaudio/default.nix

```diff
@@ -111,7 +111,9 @@ stdenv.mkDerivation rec {
     ];
 
   postInstall = lib.optionalString libOnly ''
-    rm -rf $out/{bin,share,etc,lib/{pulse-*,systemd}}
+    find $out/share -maxdepth 1 -mindepth 1 ! -name "vala" -prune -exec rm -r {} \;
+    find $out/share/vala -maxdepth 1 -mindepth 1 ! -name "vapi" -prune -exec rm -r {} \;
+    rm -rf $out/{bin,etc,lib/{pulse-*,systemd}}
```

Comment:
```suggestion
    rm -r $out/{bin,etc,lib/{pulse-*,systemd}}
```

Reply:
I've realized that this -f flag is because `$out/lib/systemd` is only present when systemd support is enabled. -f prevents the build from failing if it isn't. We can either keep the -f flag or remove it and add something like `[ -d $out/lib/systemd ] && rm -r $out/lib/systemd`.

Reply:
Can we make this conditional on systemd support?

Reply:
Continuing this discussion [here](https://github.com/NixOS/nixpkgs/pull/124913#discussion_r686495560).